### DISCUSSION TITLE
What is the weight of the loss term for each of the values in the loss_weights?

### DISCUSSION THREAD

**Discussion by baichen99** [lululxvi/deepxde#824] [Q&A] [answered]
In this [example](https://deepxde.readthedocs.io/en/latest/demos/pinn_inverse/elliptic.inverse.field.html), the loss_weights parameter is used in the `model.compile`

```python
model.compile("adam", lr=0.0001, loss_weights=[1, 100, 1000])
```

I am not quite sure which of these three weights corresponds to pde loss, boundary loss, data loss

**Comment by lululxvi** [comment, marked as answer]
https://deepxde.readthedocs.io/en/latest/demos/pinn_forward/ode.2nd.html
